UI/UI.tscn

```diff
@@ -22,7 +22,7 @@ signal player_activated(player_marker)
 
 onready var stage_label: Label = $status_view/container/StageLabel
 onready var coordinates_label: Label = $status_view/container/CoordinatesLabel
-onready var set_instance = null
+onready var placemark_instance = null
```

Comment:
i still think the variable name should mention that this variable is supposed to store the placemark thats currently being hovered